### PR TITLE
Unify checklists

### PR DESCRIPTION
**Pull request by matpfernandes**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,83 +1,114 @@
...
 ### Best Pratices
-- [ ] README
+- [ ] [README]()
 - [ ] [Dependabot]()
+
 ### Keeper
-- [ ] AWS credentials
-- [ ] Google credentials
-- [ ] Terraform env vars
+- [ ] [AWS credentials]()
```
Comment (natamvo) [inline README.md]: only the link on the keeper, the items below don't need

File: README.md [hunk 2]
```diff
@@ -1,83 +1,114 @@
...
+- [ ] [Google credentials]()
+- [ ] [Terraform env vars]()
+
 ### GDPR
-- [ ] Encrypt, pseudonymize, or anonymize personal data wherever possible
-- [ ] Sign a data processing agreement between your organization and any third parties that process personal data on your behalf
-- [ ] It's easy for your customers to request and receive all the information you have about them
-- [ ] It's easy for your customers to request to have their personal data deleted
+- [ ] [Encrypt, pseudonymize, or anonymize personal data wherever possible]()
```
Comment (natamvo) [inline README.md]: here we don't need links

File: README.md [hunk 3]
```diff
@@ -1,83 +1,114 @@
...
 ### Security
-- [ ] SSL
-- [ ] Cors
+- [ ] [SSL]()
+- [ ] [Cors]()
+
 ### CI/CD
-- [ ] CircleCI
-- [ ] All environments is deploying: lab, stg, prod (if is necessary)
+- [ ] [CircleCI]()
```
Comment (natamvo) [inline README.md]: add a comment with the example
`[CircleCI](<LINK>)`
And left the link to `https://circleci.com/docs/2.0/status-badges/#creating-badges-for-private-repositories`

File: README.md [hunk 4]
```diff
@@ -1,83 +1,114 @@
...
+- [ ] [Encrypt, pseudonymize, or anonymize personal data wherever possible]()
+- [ ] [Sign a data processing agreement between your organization and any third parties that process personal data on your behalf]()
+- [ ] [It's easy for your customers to request and receive all the information you have about them]()
+- [ ] [It's easy for your customers to request to have their personal data deleted]()
+
 ### Quality
-- [ ] CodeClimate for the project and enable Pull Requests reviews
-- [ ] Sentry is configured on Google client's account
-- [ ] pre-commit gjira installed
+- [ ] [CodeClimate for the project and enable Pull Requests reviews]()
```
Comment (natamvo) [inline README.md]: add Maintainability Badge from codeclimate

Comment (natamvo) [reply README.md]: and add the Test Coverage Badge as well as another item

File: README.md [hunk 5]
```diff
@@ -1,83 +1,114 @@
...
-- [ ] using versalite image
-- [ ] production is using guicorn
+- [ ] [using versalite image]()
+- [ ] [production is using guicorn]()
+
 ### Tests
-- [ ] Pytest configured
-- [ ] Coverage with 20% at least
+- [ ] [Pytest configured]()
+- [ ] [Coverage with 20% at least]()
```
Comment (natamvo) [inline README.md]: remove this, because is already com quality from codeclimate

File: README.md [hunk 6]
```diff
@@ -1,83 +1,114 @@
...
+
 ### Optimization
-- [ ] production is using builded folder
+- [ ] [production is using builded folder]()
+
 ### Tests
-- [ ] run test: Mocha
-- [ ] run test coverage: 20% mininum
+- [ ] [run test: Mocha]()
+- [ ] [run test coverage: 20% mininum]()
```
Comment (natamvo) [inline README.md]: remove this, because is already com quality from codeclimate

File: README.md [hunk 7]
```diff
@@ -1,83 +1,114 @@
...
+
 ### Documentation
-- [ ] Entity
-- [ ] Infra
-- [ ] API docs/swagger
+
+- [ ] [Infrastructure diagram]()
+- [ ] [Entity-Relationship diagram]()
+- [ ] [Architecture definitions document]()
+- [ ] [Sentry]()
```
Comment (natamvo) [inline README.md]: we need docs from sentry?

File: README.md [hunk 8]
```diff
@@ -1,83 +1,114 @@
...
+- [ ] [Infrastructure setup]()
+- [ ] [Updated README document]()
```
Comment (natamvo) [inline README.md]: already has one README at begin

File: README.md [hunk 9]
```diff
@@ -1,83 +1,114 @@
...
-- [ ] SSL
-- [ ] Cors
+- [ ] [SSL]()
+- [ ] [Cors]()
+
 ### CI/CD
-- [ ] CircleCI
-- [ ] All environments is deploying: lab, stg, prod (if is necessary)
+- [ ] [CircleCI]()
+- [ ] [All environments is deploying: lab, stg, prod (if is necessary)]()
```
Comment (natamvo) [inline README.md]:
```suggestion
- [ ] [All environments is deploying: lab, stg, prod (using terraform)]()
```
and add the link to terraform

File: README.md [hunk 10]
```diff
@@ -1,83 +1,114 @@
...
 ### Documentation
-- [ ] Entity
-- [ ] Infra
-- [ ] API docs/swagger
+
+- [ ] [Infrastructure diagram]()
+- [ ] [Entity-Relationship diagram]()
+- [ ] [Architecture definitions document]()
+- [ ] [Sentry]()
+- [ ] [Infrastructure setup]()
```
Comment (natamvo) [inline README.md:32]: is it about terraform?

File: README.md (hunk 4 above)
Comment (natamvo) [reply README.md]: https://codeclimate.com/github/codeclimate/codeclimate/badges

File: README.md (hunk 10 above)
Comment (matpfernandes) [reply README.md:32]: I thought about diagrams but could be the terraform config too.

**Review by natamvo — APPROVED** [review]

**Pull request merged**